docs/TRAINING.md

```diff
@@ -4,21 +4,17 @@
 
 You can use the `lumi train --model <model-type>` command to easily start training your model (with default config). Before anything else, try running it to see if everything is working. You'll need a dataset in tfrecord format in the default location (`./datasets/voc/tf`). See [DATASETS.md](./DATASETS.md) for more info.
```

Comment:
Update `lumi train`. Not sure if using `lumi train -o model.type=<model-type>` or to `lumi train -c sample_config.yml`.